dokka-integration-tests/gradle/projects/it-basic-groovy/build.gradle

```diff
@@ -14,8 +14,8 @@ dokkaHtml {
     failOnWarning = false
     dokkaSourceSets {
         customSourceSet {
-            sourceRoot(file("src/main/java"))
-            sourceRoot(file("src/main/kotlin"))
+            sourceRoot(file("src/custom/java"))
+            sourceRoot(file("src/custom/kotlin"))
```

Comment:
Same question here about why this change was needed. `custom` is less confusing than `common`, but it's still unclear what will happen if I revert / change this :)